ui/chartjs.go

```diff
@@ -0,0 +1,14 @@
+// This file is https://github.com/chartjs/Chart.js/releases/download/v2.7.1/Chart.min.js
```

Comment:
curious: any reason not to just include this as `Chart.min.js`?  And serve it directly (via http.FileServer or similar)?

Reply:
I want to be able to ‘go get istio.io/fortio‘  and have everything - do you know a standard way for go binaries to find the source tree that build them?
(That would solve the go get case but not the scp the binary around and use it)